underfs/s3a/src/test/java/alluxio/underfs/s3a/S3AUnderFileSystemTest.java

```diff
@@ -113,4 +114,23 @@ public void createCredentialsFromDefault() throws Exception {
         S3AUnderFileSystem.createAwsCredentialsProvider(ufsConf);
     Assert.assertTrue(credentialsProvider instanceof DefaultAWSCredentialsProviderChain);
   }
+
+  @Test
+  public void getPermissionsCached() throws Exception {
+    Mockito.when(mClient.getS3AccountOwner()).thenReturn(new Owner("0", "test"));
+    Mockito.when(mClient.getBucketAcl(Mockito.anyString())).thenReturn(new AccessControlList());
+    mS3UnderFileSystem.getPermissions();
+    mS3UnderFileSystem.getPermissions();
+    Mockito.verify(mClient).getS3AccountOwner();
+    Mockito.verify(mClient).getBucketAcl(Mockito.anyString());
+  }
+
+  @Test
+  public void getPermissionsDefault() throws Exception {
+    Mockito.when(mClient.getS3AccountOwner()).thenThrow(AmazonClientException.class);
+    ObjectUnderFileSystem.ObjectPermissions permissions = mS3UnderFileSystem.getPermissions();
+    Assert.assertEquals("", permissions.getGroup());
+    Assert.assertEquals("", permissions.getOwner());
+    Assert.assertEquals(0700, permissions.getMode());
```

Comment:
I changed it to be a constant in the test, I think we could also make the field protected in `S3AUnderFileSystem` but we don't do that in general (ie. visible for testing).